fabric-client/lib/Client.js

```diff
@@ -562,7 +562,7 @@ const Client = class extends BaseClient {
 	_buildCAfromConfig(ca_info) {
 		let tlsCACerts = ca_info.getTlsCACerts();
 		if (tlsCACerts) {
-			tlsCACerts = [tlsCACerts];
+			// use it as is
```

Comment:
Should we test to see if it's an array first? Then we can support both string and array. E.g.
```
if(!Array.isArray(tlsCACerts)) {
   tlsCACerts = [tlsCACerts];
} // otherwise use it as it is
```